### PR TITLE
Add UPS client (netclient) support in NUT plugin

### PR DESCRIPTION
**Pull request by daniel0076**
Fixes: #318
The patch is for OMV4 (the one for OMV5 WIP)

Signed-off-by: Daniel Tsai <idaniel.twc@gmail.com>

- [x] References issue
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by votdev** [comment]
I'm sorry, but OMV4 does not get any new features anymore. Feature freeze was long time ago. This version only gets bugfixes.

File: deb/openmediavault-nut/var/www/openmediavault/js/omv/module/admin/service/nut/Nut.js [hunk 1]
```diff
@@ -77,6 +114,28 @@ Ext.define("OMV.module.admin.service.nut.Settings", {
...
 				checked: false
+			},{
+				xtype: "combo",
+				name: "mode",
+				fieldLabel: _("NUT mode"),
+				queryMode: "local",
+				store: Ext.create("Ext.data.ArrayStore", {
+					fields: [ "value", "text" ],
+					data: [
+						[ "netclient", _("netclient") ],
```
Comment (votdev) [inline deb/openmediavault-nut/var/www/openmediavault/js/omv/module/admin/service/nut/Nut.js]:
```suggestion
						[ "netclient", _("Netclient") ],
```

File: deb/openmediavault-nut/var/www/openmediavault/js/omv/module/admin/service/nut/Nut.js [hunk 2]
```diff
@@ -77,6 +114,28 @@ Ext.define("OMV.module.admin.service.nut.Settings", {
...
+						[ "standalone", _("standalone") ],
```
Comment (votdev) [inline deb/openmediavault-nut/var/www/openmediavault/js/omv/module/admin/service/nut/Nut.js]:
```suggestion
						[ "standalone", _("Standalone") ],
```

File: deb/openmediavault-nut/var/www/openmediavault/js/omv/module/admin/service/nut/Nut.js [hunk 3]
```diff
@@ -77,6 +114,28 @@ Ext.define("OMV.module.admin.service.nut.Settings", {
...
+				}),
+				displayField: "text",
+				valueField: "value",
+				allowBlank: false,
+				editable: false,
+				triggerAction: "all",
+				value: "standalone",
+				plugins: [{
+					ptype: "fieldinfo",
+					text: _("'standalone' for local UPS, 'netclient' for remote UPS on a NUT server"),
```
Comment (votdev) [inline deb/openmediavault-nut/var/www/openmediavault/js/omv/module/admin/service/nut/Nut.js]:
```suggestion
					text: _("'Standalone' for local UPS, 'Netclient' for remote UPS on a NUT server"),
```

**Comment by daniel0076** [comment]
Okay, thanks for the reply and suggestions.
Working on OMV5 version.

**Pull request closed without merging**